src/handlers/motions/motionStaked/helpers.ts

```diff
@@ -182,3 +182,71 @@ export const getUserMinStake = (
     .div(BigNumber.from(10).pow(36))
     .toString();
 };
+
+interface Props {
+  motionData: MotionData,
+  messages: MotionMessage[];
+  requiredStake: BigNumber;
+  motionStakes: MotionStakes,
+  messageKey: string,
+  vote: BigNumber,
+  staker: string,
+  amount: BigNumber,
+}
+
+export const getUpdatedMessages = ({
+  motionData,
+  messages,
+  requiredStake,
+  motionStakes,
+  messageKey,
+  vote,
+  staker,
+  amount,
+}: Props): MotionMessage[] => {
+  const updatedMessages = [...messages];
+  const isFirstObjection = vote.eq(MotionVote.NAY) && !motionData.hasObjection;
+  if (isFirstObjection) {
+    motionData.hasObjection = true;
+    updatedMessages.push(
+      {
+        name: 'ObjectionRaised',
```

Comment:
These strings should come from an enum, like in the fe

Reply:
great idea, thanks